### PR TITLE
👷 JAVA-3134 Replace Travis CI with GitHub Actions

### PR DESCRIPTION
**Pull request by gilday**
* Introduces Maven Wrapper to pin Maven at version 3.8.1
* Updates README to include Maven Wrapper
* Updates README to lead users to Maven Central instead of building the project locally
* Replaces Travis CI with GitHub Actions

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (bryanateoan) [inline .mvn/wrapper/MavenWrapperDownloader.java:1]: Is this comment block the "Notice" required for redistribution?

**Review by bryanateoan — CHANGES_REQUESTED** [review]
1. Add back in the .project and .settings created by eclipse
2. Question about apache 2.0

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (gilday) [reply .mvn/wrapper/MavenWrapperDownloader.java:1]: Yes, this file is generated by the [Maven Wrapper](https://github.com/takari/maven-wrapper) and is intended to be checked into vcs

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,195 @@
-.idea/*
+# Project
+/.idea
 *.iml
-/target
-/.classpath
-/.project
-/.settings/
```
[the remark this answers is not in the document]
Comment (gilday) [reply .gitignore:6]: Good catch. All of the edits I've made here are from https://github.com/github/gitignore, so I've added the [Eclipse ignore file](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore)

File: README.md [hunk 3]
```diff
@@ -55,4 +60,6 @@ WebGoat (48K LOC)
 WebGoat7 (106K LOC)
 ```
 
+## Building
 
+Use `./mvn verify` to build and test changes to the project
```
Comment (bryanateoan) [inline README.md:65]: `./mvnw verify`

**Review by bryanateoan — CHANGES_REQUESTED** [review]
1. change to `./mvnw verify` in readme.

Rest of the changes looks good.

File: README.md (hunk 3 above)
Comment (bryanateoan) [reply README.md:65]: consider `./mvnw verify clean install` instead so that the developer remembers to install.

Comment (gilday) [reply README.md:65]: Are you sure users need to `install`? If they're just trying to test their changes to this project, it should be unnecessary. The Maven community has been trying to re-educate users on when they actually need to `clean` and `install`, and their message is that most of the time you just need `mvn verify` 

* https://twitter.com/ASFMavenProject/status/1200844862897172480
* https://twitter.com/ASFMavenProject/status/1002840059337412608

Comment (bryanateoan) [reply README.md:65]: yeah. `verify` is probably sufficient.

**Review by stiemannkj1 — APPROVED** [review]
Looks good to me. I think we should move to the latest version of Maven Wrapper ([from ASF](https://github.com/apache/maven-wrapper-plugin)) if possible, but if you think that’s out-of-scope, then we can leave as-is.

File: .mvn/wrapper/maven-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.1/apache-maven-3.8.1-bin.zip
+wrapperUrl=https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar
```
[the remark this answers is not in the document]
Comment (gilday) [reply .mvn/wrapper/maven-wrapper.properties:2]: I tried, but the ASF wrapper is brand new and only supports Maven 4.x (which has not yet been released). 🔜

**Review by BrianPhillips2020 — APPROVED** [review]

**Review by aliaandersen — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (gilday) [reply README.md:65]: Cool. @bryanateoan are you still waiting on me to make changes somewhere? Your review is still in "Changes Requested" state

Comment (bryanateoan) [reply README.md:65]: can you change the `./mvn verify` to `./mvnw verify`?

Comment (gilday) [reply README.md]: Ah I didn't see that change. Pushed

**Review by bryanateoan — APPROVED** [review]

**Pull request merged**